GOVERNANCE.md

```diff
@@ -360,7 +360,7 @@ If the pull request submitter is not a Core Member, the PR can be closed by any
 
 Astro features are discussed using a model called [Consensus-seeking decision-making](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making). This model attempts to achieve consensus on all significant changes to Astro, but has a fallback voting procedure in place if consensus appears unattainable.
 
-**Who can vote:** All [TSC](#technical-steering-committee-tsc) members.
+**Who can vote:** All [TSC](#technical-steering-committee-tsc) members and Core members.
```

Comment:
Do we need to mention TSC at all since it's a subset of core?

Reply:
I don't want to take responsibility for adding more changes, as this could lead to a longer discussion.

Reply:
I feel like we can remove this line entirely, the list below this doesn't mention there even being a vote. There really *isn't* a vote.